wgengine/router/router_linux_test.go

```diff
@@ -19,8 +19,8 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/tailscale/netlink"
 	"github.com/tailscale/wireguard-go/tun"
-	"github.com/vishvananda/netlink"
```

Comment:
I swapped this with `"github.com/tailscale/netlink"` because my newly added test needed the same types from `router_linux.go`. I'm not sure why `"github.com/vishvananda/netlink"` was ever used. I'm calling out in case this is a problem. The tests still pass locally.